source/_integrations/miele.markdown

```diff
@@ -66,6 +66,10 @@ Internal examples: `http://192.168.0.2:8123/auth/external/callback`, `http://hom
 
 {% enddetails %}
 
+## Local discovery
```

Comment:
it's too far away for me to make a suggestion. but we usually add this to the header:
ha_zeroconf: true